projects/aca-shared/rules/src/app.rules.ts

```diff
@@ -25,11 +25,61 @@
 
 import { AppConfigService } from '@alfresco/adf-core';
 import { RuleContext } from '@alfresco/adf-extensions';
-import { getFileExtension, supportedExtensions } from '@alfresco/adf-office-services-ext';
```

Comment:
`aca-shared` should not depend on other libraries (`adf-office-services-ext` content-ce lib, do not mistake it  with lib from adf : )

e.g. `projects/adf-office-services-ext/src/lib/aos-extension.module.ts` imports `aca-shared` what creates a circular dependancies.